unlock-app/src/components/content/event/EventDetails.tsx

```diff
@@ -139,7 +139,9 @@ export const EventDetails = ({ lockAddress, network }: EventDetailsProps) => {
           {eventDate && eventData.ticket?.event_start_time && (
             <li className="mb-2">
               <FaClock className="inline mr-2" />
-              {eventDate.toLocaleTimeString()}
+              {eventDate.toLocaleTimeString(navigator.language || 'en-US', {
+                timeZone: eventData.ticket.event_timezone,
+              })}
```

Comment:
Do we hint that we are using event location timezone than user timezone? If not, we should.

Reply:
Good point. Let me add a hint!